April - May update (#176)
ilios/iliosproject.org: _posts/2021-04-19-updates.markdown

---
layout: post
title: Gone is April, Come what May
date: 2021-04-19  08:00:00
categories: updates
---

![IliosInSpringImg](https://mcusercontent.com/845c4ebabb5b5ae7a6372c715/images/e1f0e196-2298-434d-ba15-8f94a0fa12c8.png)

>  A new season, a new month, and here at Ilios we are rolling ahead into May with plans to continue our current development efforts, as we work on identifying areas for improvement in reporting and display, including a look at ways to provide an interactive "checklist" for activities and materials so learners can track their efforts. You can expect a number of updates in these areas in the coming months, so keep eyes open for better reporting, improved visualization, and increased interactive tools!

Reviewer: That first sentence is quite long.

Beyond that, some minor suggestions, only one of which I would consider an error being fixed (which is the missing pronoun before _eyes_):

```suggestion
A new season, a new month, and here at Ilios, we are rolling ahead into May with plans to continue our current development efforts, as we work on identifying areas for improvement in reporting and display, including a look at ways to provide an interactive "checklist" for activities and materials so learners can track their efforts. You can expect several updates in these areas in the coming months, so keep your eyes open for better reporting, improved visualization, and increased interactive tools!
```
